tests/test_mdlp.py

```diff
@@ -32,165 +34,31 @@ def test_find_cut_no_cut():
     k = find_cut(y, 0, len(y))
     assert_equal(-1, k)
 
-def test_mdlp_iris():
-    from sklearn.datasets import load_iris
+def test_fit_transform():
+  X = np.array([
+    [0.1, 0.1],
+    [0.2, 0.4],
+    [0.3, 0.2],
+    [0.4, 0.3]
+  ])
+  y = np.array([0, 0, 1, 2])
+  transformed = MDLP(shuffle=False).fit_transform(X, y)
+  expected = [
+    [0, 0],
+    [0, 0],
+    [1, 0],
+    [2, 0],
+  ]
+  assert_array_equal(transformed, expected)
```

Comment:
For testing APIs, the `expected` variable generally comes before the `actual` value.

Reply:
ok. what does 'generally' mean?